src/components/modules/dashboard/marketplace/ui/dialogs/MarketplaceLoanDetailDialog.tsx

```
@@ -46,111 +58,310 @@ const MarketplaceLoanDetailDialog = ({
+            value="milestones"
+            className="p-6 pt-4 focus-visible:outline-none focus-visible:ring-0"
+          >
+            <div className="space-y-4">
+              <div className="flex items-center gap-2">
+                <Milestone className="h-5 w-5 text-emerald-600" />
+                <h3 className="font-medium">Loan Milestones</h3>
+              </div>
+
+              {selectedLoan.milestones?.length > 0 ? (
+                <div className="space-y-3">
+                  {selectedLoan.milestones?.map((m: any, idx: number) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)